testhelpers/net.go

```diff
@@ -122,6 +125,17 @@ func RandPeerID() (peer.ID, error) {
 	return peer.ID(h), nil
 }
 
+// RequireIntPeerID takes in an integer and creates a unique peer id for it.
+func RequireIntPeerID(t *testing.T, i int64) peer.ID {
```

Comment:
I would prefer using this pattern for peerId generation, your call.
https://github.com/filecoin-project/go-filecoin/blob/72652f1c1336d091d08252899b0ff5872dd5a459/address/testing.go#L7-L20